pt.html

```diff
@@ -302,44 +309,49 @@
 <panel w=550 h=450>
 <pic src="pics/intro0.png" sx=950 sy=1600></pic>
 <words x=10 y=10 w=480 h=90>
-	Still, isn't "rote memorization" bad?
-	Can't we look everything up these days?
-	Shouldn't we learn creativity &amp; critical thinking instead?
+	Opa, mas o "habito de memorizar" não é ruim?
```

Comment:
Acho que habito de memorizar não condiz com o termo de rote memorization né?
```suggestion
	Opa, mas "memorização mecânica" não é ruim?
```

Reply:
Hummmmmmm, pelo visto você diz isso aqui https://en.wikipedia.org/wiki/Rote_learning. Mais tarde quando acordar respondo com calma.

Pior que vários destes termos não tem nem um draft na wikipedia. Até comentei lá no post original com o tradudor pra alemão de que talvez valesse a pena rascunhas as respectivas páginas em breve.